frontend/tests/controllers/UserRegistrationTest.php

```diff
@@ -186,4 +186,33 @@ public function testUserParentalTokenNotGeneratedDueInvalidParameters() {
             $this->assertSame('parameterInvalid', $e->getMessage());
         }
     }
+
+    /**
+     * User 13 and its account link to the parent account
+     * on verification of parental token
+     */
+    public function testUse13LinkedToParentAccountWhenTokenVerificationDone() {
+        $defaultDate = strtotime('2022-01-01T00:00:00Z');
+        \OmegaUp\Time::setTimeForTesting($defaultDate);
+        // Create a 10 years-old user
+        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser(
```

Comment:
Here you could use `User` object instead of `Identity`
```suggestion
        ['user' => $user] = \OmegaUp\Test\Factories\User::createUser(
```